### DISCUSSION TITLE
Is it still worth contributing changes to the current Northern Rebirth?

### DISCUSSION THREAD

**Discussion by cooljeanius** [nemaara/Northern_Rebirth_Reborn#5] [Q&A] [answered]
I was thinking about doing some more changes to [my current NR PR](https://github.com/wesnoth/wesnoth/pull/6057), and eventually lifting it out of draft mode; is that still worthwhile? Or should I just give up on it now?

**Comment by nemaara** [comment, marked as answer]
That's entirely up to you. At this moment it looks very likely that we will be able to get a working version in time for 1.18 but it's not guaranteed.
